File: MinSum.py
class Solution:
    def solve(self, arr, n):
        # code here
        arr.sort()
        s1 = "" ; 
        s2 = "" ; 
        for i in range(n):
            if i%2==0:
                s1+=str(arr[i])
            else:
                s2+=str(arr[i])
        if s1=="":
            s1="0"
        if s2=="":
            s2="0"
        return int(s1) + int(s2)

File: MinSum.java
import java.util.Arrays;

public class MinSum {

          public long minValue(long a[], long b[], long n) 
    {
        // Your code goes here
        Arrays.sort(a);
        Arrays.sort(b) ; 
        
        int i = 0 ; 
        int j = b.length -1;
        long sum = 0 ;
        while(i<n && j>=0){
            sum+=a[i]*b[j] ; 
            i++;
            j-- ; 
        }
        return sum ; 
    }
      
}
